diff --git a/node_modules/is-porn/lib/is-porn.js b/node_modules/is-porn/lib/is-porn.js
index f18a94b..e6752ef 100644
--- a/node_modules/is-porn/lib/is-porn.js
+++ b/node_modules/is-porn/lib/is-porn.js
@@ -2,8 +2,8 @@
 const nslookup = require('nslookup');
 const debug = require('debug')('is-porn');
 const OPENDNS_IPS = [
-    /67\.215\.65.\d{3}/gi,
-    /146\.112\.61\.\d{3}/gi,
+    /67\.215\.65.\d{3}/i,
+    /146\.112\.61\.\d{3}/i,
 ];
 
 var isOpenDNS = function(ip) {
